doctr/file_utils.py

```diff
@@ -10,13 +10,16 @@
 import os
 import sys
 
+CLASS_NAME: str = "words"
```

Comment:
I think it is ok to have this default single class and use the class of the datasets otherwise to build the model, but the user should be able to modify this CLASS_NAME to a specific value like "words", "numbers", "default_class", "handwritten"... so it shouldn't be hardcoded here but I rather see it as a parameter given by the user either in the config or when building the mode, WDYT ?

Reply:
Hi @charlesmindee yep sounds good to me 👍 

I see it much more problematic to modify the blocks output structure .. the results should be added to each predicted word like

'house', conf: '0.99', class: 'word'

Instead of:

Blocks: 
'word':  { Line .. Words ..},
'salary': { Line .. Words ..}

Wdyt ?